lib/Remotery.c

```diff
@@ -3385,6 +3399,7 @@ static rmtError Sample_Constructor(Sample* sample)
 
 static void Sample_Destructor(Sample* sample)
 {
+    assert(sample == 0 || sample != 0);	// To fool warning C4100 on warning level 4
     RMT_UNREFERENCED_PARAMETER(sample);
 }
 
```

Comment:
I could move it in to the RMT_UNREFERENCED_PARAMETER define as "if MIcrosoft then use the assert" ?